### PR TITLE
implemented meta params on actions: create, update, remove

### PR DESCRIPTION
**Pull request by damirjuretic**

### CODE REVIEW

File: src/actions/create.js [hunk 1]
```diff
@@ -18,7 +20,7 @@ import thunkAction from './_thunkAction';
  * @param item holds object that you want to pass to api
  * @returns {function}
  */
-export function create(config, schema, item = null) {
+export function create(config, schema, item = null, params = {}, options = {}) {
```
Comment (vedrani) [inline src/actions/create.js]: signature should be `create(schema, params = {}, options = {})` and schema should be resolved by name or by object same as in `find`

File: src/actions/create.js [hunk 2]
```diff
@@ -39,16 +41,25 @@ export function create(config, schema, item = null) {
     body = config.body;
   }
 
+  const isEndpointResolved = options[RESOLVED_ENDPOINT];
+  const endpoint = isEndpointResolved
+    ? config.endpoint
+    : buildEndpoint(config.endpoint, params);
+
   const meta = {
```
Comment (vedrani) [inline src/actions/create.js]: missing part from resolved config

File: src/actions/create.js [hunk 3]
```diff
@@ -39,16 +41,25 @@ export function create(config, schema, item = null) {
     body = config.body;
   }
 
+  const isEndpointResolved = options[RESOLVED_ENDPOINT];
+  const endpoint = isEndpointResolved
```
Comment (vedrani) [inline src/actions/create.js]: also use `buildRSAAConfig`

Comment (vedrani) [inline src/actions/create.js]: those shared parts like `buildRSAAConfig` chould be moved into seperate file, for now maybe schemaConfig.js

File: src/actions/update.js [hunk 4]
```diff
@@ -4,45 +4,62 @@ import {
...
   }
   if (!_.isObject(item)) {
     throw new Error('Item isn\'t object.');
   }
 
+  const rsaaConfig = buildRSAAConfig(config);
+  const isEndpointResolved = options[RESOLVED_ENDPOINT];
+  const endpoint = isEndpointResolved
+    ? rsaaConfig.endpoint
+    : buildEndpoint(rsaaConfig.endpoint, params);
```
Comment (vedrani) [inline src/actions/update.js]: It should be like
```
a ?
b :
c;
```

Apply to all cases in actions find,update,remove, create

File: test/actions/update.spec.js [hunk 5]
```diff
@@ -61,30 +70,75 @@ describe('Update action creator', () => {
...
+      timestamp: types[0].meta.timestamp,
+    };
+    expect(types[0].type).to.equal(UPDATE_REQUEST);
+    expect(types[0].meta).to.deep.equal(expectedMeta);
+    expect(types[1].type).to.equal(UPDATE_SUCCESS);
+    expect(types[1].meta).to.deep.equal(expectedMeta);
   });
 
-  it('throws exception on invalid action with invalid schema', () => {
+  it('throws exception on action with schema configuration is invalid', () => {
```
Comment (vedrani) [inline test/actions/update.spec.js]: Add tests in create, update remove for case when schema is undefined.

File: src/actions/update.js (hunk 4 above)
Comment (MrBr) [inline src/actions/update.js]: This should be a function, same logic is used and above at https://github.com/shoutem/redux-io/pull/2/files#diff-878372f8bd49ae8e22b4063be10afbc5R44

**Pull request merged**